go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ONSdigital/dp-recipe-api
 
-go 1.18
+go 1.17
```

Comment:
Why did we reduced the go version?

Reply:
I'm not sure, this was one of @ChrysmOre commits

Reply:
Ok. I am going to approve it, as it is very low risk, but the CI and the environments rely on 1.18 and not 1.17.
This was probably a dependency or a security issue.
@ChrysmOre it would be good to understand this and bring it back to 1.18.